phase1/gce/gce.jsonnet

```diff
@@ -149,7 +149,7 @@ function(cfg)
             "k8s-apisever-private-key": "${tls_private_key.%s-master.private_key_pem}" % p1.cluster_name,
             "k8s-master-kubeconfig": kubeconfig(p1.cluster_name + "-master", "local", "service-account-context"),
             "k8s-advertise-addresses": "${google_compute_address.%(master_ip)s.address}" % names,
-            "k8s-cni-plugin": p3.cni,
+            "k8s-cni-plugin": if std.objectHas(p3, "cni") then p3.cni else "undefined",
```

Comment:
Using the string `undefined` feels a little dirty here. How about just an empty string?